app/assets/javascripts/vote.js

```diff
@@ -0,0 +1,49 @@
+var makeAjaxRequestUpPost = function(reviewButtonTo, venueID, reviewId, voteType) {
+  var request = $.ajax({
+    method: 'PATCH',
+    data: { up: voteType },
+    url: '/api/venues/' + venueID + '/reviews/' + reviewId
+  });
+
+  request.done(function(data) {
+    var review = $('#review-' + data.id)[0]
+    review.getElementsByClassName("helpful")[0].innerHTML = "Helpful: " + data.revup_count
+    review.getElementsByClassName("not-helpful")[0].innerHTML = "Not Helpful: " + data.revdown_count
+  });
+};
+
+var makeAjaxRequestDownPost = function(reviewButtonTo, venueID, reviewId, voteType) {
+  var request = $.ajax({
+    method: 'PATCH',
+    data: { down: voteType },
+    url: '/api/venues/' + venueID + '/reviews/' + reviewId
+  });
+
+  request.done(function(data) {
+    var review = $('#review-' + data.id)[0]
+    review.getElementsByClassName("helpful")[0].innerHTML = "Helpful: " + data.revup_count
+    review.getElementsByClassName("not-helpful")[0].innerHTML = "Not Helpful: " + data.revdown_count
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Identifier 'revdown_count' is not in camel case.<br>Missing semicolon.